pyproject.toml

```diff
@@ -39,7 +39,7 @@ known_third_party = ["diastatic-malt", "jax", "jaxlib", "numpy", "pennylane"]
 skips = ["B607"]
 
 [build-system]
-requires = ["setuptools>=62", "wheel", "pybind11>=2.12.0", "numpy!=2.0.0"]
+requires = ["setuptools>=62", "wheel", "pybind11>=2.12.0", "numpy!=2.0.0", "nanobind", "cmake", "ninja"]
```

Comment:
Do we still require pybind then? (or do you plan to remove all pybind installations after all PRs are merged?)

Reply:
Yes I was planning to remove all references to pybind11 in a separate PR once all of the other PRs are merged.